### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Bondok6**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
-# yat_project
+# Yat Project
 
-Link: https://yat-project.herokuapp.com/
+Live Demo Link: https://yat-project.herokuapp.com/
```
Comment (Reem-lab) [inline README.md:3]: you have to changes this Kyrillos

**Review by Reem-lab — CHANGES_REQUESTED** [review]
## Requested Changes 💔

File: README.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
-# yat_project
```
Comment (Reem-lab) [inline README.md:1]: you have to

**Review by Reem-lab — COMMENTED** [review]
nvn

File: README.md (hunk 2 above)
Comment (Bondok6) [reply README.md:1]: 5alsa

**Pull request closed without merging**